pkgs/by-name/ic/icomoon-feather/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchgit,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "icomoon-feather";
+  version = "2024-05-11";
+
+  src = fetchgit {
```

Comment:
Unable to use `fetchFromGitHub`?

Reply:
I was facing some issue while doing `sparseCheckout`, so I fell back on `fetchgit`